pkg/runtime/ports.go

```
@@ -0,0 +1,62 @@
+	}
+
+	return ports
+}
+
+// IsPortFree checks whether a specific TCP port on 127.0.0.1 is available.
+func IsPortFree(port int) bool {
+	l, err := net.Listen("tcp", fmt.Sprintf("127.0.0.1:%d", port))
+	if err != nil {
+		return false
+	}
+	l.Close()
```

golangci-lint (2.12.1)

[error] Error return value of `l.Close` is not checked

(errcheck)